tests/import/import_test.bzl

```diff
@@ -3,7 +3,22 @@ load("@rules_sh//sh:import.bzl", "create_shim")
 
 # create shim ########################################################
 
+def _running_workspace():
```

Comment:
Unfortunately, detecting `bzlmod` by creating a dummy label and checking if there is a `@@` prefix doesn't work on the CI (but works locally).

Reply:
That's surprising, and shouldn't really be the case. This may indicate a configuration issue. E.g. I see that the workspace builds [correctly enable BuildBuddy](https://github.com/tweag/rules_sh/actions/runs/4153648344/jobs/7185406101#step:6:82), but the bzlmod ones [don't seem to](https://github.com/tweag/rules_sh/actions/runs/4153648344/jobs/7185406192#step:5:73).
I notice that the CI script copies `.bazelrc.local` into `tests/`, but there's no corresponding `.bazelrc`. Bazel only looks for [`.bazelrc` in the current working directory and some global locations](https://bazel.build/run/bazelrc). But, `.bazelrc.local` is only considered because of the line `try-import %workspace%/.bazelrc.local` in `.bazelrc`.
So, I think these builds don't actually run with bzlmod enabled on CI.

Reply:
Indeed that was an issue. I've updated the workflow to also copy over the `.bazelrc` and `.bazelrc.local` files.